### DISCUSSION TITLE
Search within pre-existing list view

### DISCUSSION THREAD

**Discussion by joleneb22** [microsoft-search/pnp-modern-search#2660] [Q&A]
We have a list with a pre-existing view used to restrict what people can see. The view filters are:

VendorLogin = [Me] OR VendorLogin2 = [Me] OR VendorLogin3 = [Me]

Is there way to easily filter a search results page based on this pre-existing view, or how would we apply similar filter to the search query based on the List address?

**Comment by gretchunkim** [comment]
Is the list view with path (URL) defined? I have similar case and I use the following in query template: 

 {searchTerms} path: "url of the particular view, in my case it's SharePoint list view" and it seems to work.

**Comment by wobba** [comment]
@joleneb22 if your list has site columns you will have automatically generated managed properties you can use in a query template to achieve the same filtering. If you don't have site columns you need to map the crawled properties to a reusable manage property and use that one instead.
